### ISSUE TITLE
look like not being trained

### ISSUE THREAD

**Issue by CUN-bjy** [CUN-bjy/gym-ddpg-keras#4]
after 20 episodes, the reward 's so poor.

```bash
=========EPISODE # 16 ==========
100%|########################################9| 499/500 [00:53<00:00,  8.67it/s]Episode#16, steps:499, rewards:-311.104067
100%|########################################9| 499/500 [00:53<00:00,  9.28it/s]
=========EPISODE # 17 ==========
100%|########################################8| 498/500 [00:51<00:00, 10.16it/s]Episode#17, steps:499, rewards:-211.220928
100%|########################################9| 499/500 [00:51<00:00,  9.67it/s]
=========EPISODE # 18 ==========
100%|########################################8| 498/500 [00:55<00:00,  9.79it/s]Episode#18, steps:499, rewards:-243.705719
100%|########################################9| 499/500 [00:56<00:00,  8.90it/s]
=========EPISODE # 19 ==========
100%|########################################9| 499/500 [00:55<00:00,  7.42it/s]Episode#19, steps:499, rewards:-332.657685
100%|########################################9| 499/500 [00:56<00:00,  8.90it/s]
=========EPISODE # 20 ==========
100%|########################################9| 499/500 [00:55<00:00,  9.19it/s]Episode#20, steps:499, rewards:-231.899928
```

**Comment by CUN-bjy** [comment]
```bash
=========EPISODE # 102 ==========
100%|########################################9| 499/500 [01:03<00:00,  6.45it/s]Episode#102, steps:499, rewards:-303.195322
100%|########################################9| 499/500 [01:03<00:00,  7.83it/s]
=========EPISODE # 103 ==========
100%|########################################9| 499/500 [01:04<00:00,  8.09it/s]Episode#103, steps:499, rewards:-349.769675
100%|########################################9| 499/500 [01:04<00:00,  7.78it/s]
=========EPISODE # 104 ==========
100%|########################################9| 499/500 [01:03<00:00,  7.33it/s]Episode#104, steps:499, rewards:-257.166998
100%|########################################9| 499/500 [01:04<00:00,  7.79it/s]
=========EPISODE # 105 ==========
100%|########################################9| 499/500 [01:09<00:00,  6.28it/s]Episode#105, steps:499, rewards:-104.307859
100%|########################################9| 499/500 [01:09<00:00,  7.21it/s]
```

**Comment by CUN-bjy** [comment]
On Episode #66
the action holds on some values.
```bash
w/o noise [ 1.  1.  1. -1.  1. -1. -1.  1.]
w/ noise [ 1.          1.          0.26686811 -1.          0.45160865 -1.
 -0.52892347  0.89307425]
 47%|██████████████████▎                    | 234/500 [00:25<00:29,  8.98it/s]
w/o noise [ 1.  1.  1. -1.  1. -1. -1.  1.]
w/ noise [ 1.          1.          0.26796781 -1.          0.45243124 -1.
 -0.52963008  0.89323464]
 47%|██████████████████▎                    | 235/500 [00:25<00:29,  9.05it/s]
w/o noise [ 1.  1.  1. -1.  1. -1. -1.  1.]
w/ noise [ 1.          1.          0.26906585 -1.          0.45325259 -1.
 -0.53033564  0.89339479]
```

**Comment by CUN-bjy** [comment]
I Changed training functions referenced by [this](https://github.com/marload/DeepRL-TensorFlow2/blob/master/DDPG/DDPG_Continuous.py)

And I think this code is simliar to the original paper.
(the code use Q gradients when training for actor)

**Comment by CUN-bjy** [comment]
Eventhogh, the model isn't going to be optimized.

I print the critic's loss value.
```bash 
critic loss : 6.72760534
 99%|████████████████████████████████████████▌| 494/500 [00:56<00:00,  8.90it/s]critic loss : 6.31205
 99%|████████████████████████████████████████▌| 495/500 [00:56<00:00,  8.93it/s]critic loss : 70.4440384
 99%|████████████████████████████████████████▋| 496/500 [00:56<00:00,  8.96it/s]critic loss : 2.40150857
 99%|████████████████████████████████████████▊| 497/500 [00:56<00:00,  9.05it/s]critic loss : 8.08836174
100%|████████████████████████████████████████▊| 498/500 [00:57<00:00,  9.07it/s]critic loss : 4.9618187
100%|██████████████████████████████████████
```

the loss value was 0.0x at the 1st episode.
but after 20 epi more, the loss value was going high.
something wrong... the critic is not trained.

**Comment by CUN-bjy** [comment]
When I changed `tau` parameter 0.001 to 0.1, the loss was going high in faster speed.
That means the training function is wrong, couldn't fit the model.

**Comment by CUN-bjy** [comment]
Maybe..
The problem is, Exploration.
